tool/jt.rb

```diff
@@ -157,11 +157,19 @@ def self.find_sl
     raise "couldn't find truffle-sl.jar - build Truffle and find it in there"
   end
 
-  def self.find_launcher
-    if ENV['RUBY_BIN']
-      ENV['RUBY_BIN']
+  def self.find_launcher(use_svm)
+    if use_svm
+      if ENV['AOT_BIN']
+        ENV['AOT_BIN']
+      else
+        "#{TRUFFLERUBY_DIR}/bin/native-ruby"
+      end
```

Comment:
I think `ENV['AOT_BIN'] || "#{TRUFFLERUBY_DIR}/bin/native-ruby"` or `ENV.fetch()` is simpler.

Reply:
Fair enough. I just matched what was being done for `RUBY_BIN`.